code/modules/clothing/clothing.dm

```diff
@@ -216,6 +216,34 @@ BLIND     // can't see anything
 
 	..()
 
+/obj/item/clothing/under/CtrlClick(mob/user)
+	if(!istype(user) || !user.canUseTopic(src, BE_CLOSE, ismonkey(user)))
+		return
+
+	var/mob/M = user
+	if (istype(M, /mob/dead/))
+		return
+	if (!can_use(M))
+		return
+	if(src.has_sensor == LOCKED_SENSORS)
+		to_chat(usr, "The controls are locked.")
+		return 0
+	if(src.has_sensor == BROKEN_SENSORS)
+		to_chat(usr, "The sensors have shorted out!")

```

Comment:
to_chat(user, [...]

Proof of copypasta. But since this is not a verb, so use the defined user var instead.